src/simplewallet/simplewallet.cpp

```diff
@@ -3018,6 +3099,12 @@ bool simple_wallet::save_watch_only(const std::vector<std::string> &args/* = std
 //----------------------------------------------------------------------------------------------------
 bool simple_wallet::start_mining(const std::vector<std::string>& args)
 {
+  if (m_wallet->key_on_device())
+  {
+    fail_msg_writer() << tr("command not supported by HW wallet");
+    return true;
+  }
+
```

Comment:
You don't need any secret to mine, just the standard address.

Reply:
ok, give it a try :)

Reply:
Forgot to delete?